src/YiiElasticSearch/MultiSearchDataProvider.php

```diff
@@ -0,0 +1,133 @@
+<?php
+namespace YiiElasticSearch;
+
+/**
+ * A Data provider for searching into multiple indices
+ */
+class MultiSearchDataProvider extends DataProvider
+{
+    public $connID = 'elasticSearch';
+    public $models = array();
```

Comment:
@tavantzo Can you document that  a little? Actually I'm not really sure if I understand how is is going to be used. Especially this line below:

```
$reflObject = new \ReflectionObject($this->models["{$result->getIndex()}-{$result->getType()}"]);
```

It seems like you expect some quite complex keys in this array here. Could you explain?